osu.Game.Rulesets.Osu/Skinning/Default/SpinnerRotationTracker.cs

```diff
@@ -77,6 +78,10 @@ protected override void Update()
             Rotation = (float)Interpolation.Damp(Rotation, currentRotation, 0.99, Math.Abs(Time.Elapsed));
         }
 
+        private float currentDelta;
```

Comment:
Rename to "deltaSinceLastDirectionChange"? If this name seems correct then it may be enough to document what this is. Else xmldoc would be required.